website/source/docs/guides/agent-encryption.html.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: "docs"
+page_title: "Agent Communication Encryption"
+sidebar_current: "docs-guides-agent-encryption"
+description: |-
+  This guide covers how to encrypt both gossip and RPC communication.
+---
+
+# Agent Communication Encryption
+
+There are two separate systems that need to be configured separately to encrypt communication within the cluster, gossip encryption and TLS. TLS is used to secure the RPC calls between agents. Gossip encryption is secured with a symmetric key, since gossip between nodes is done over UDP. In this guide we will configure both.
```

Comment:
It feels a bit repetitive to say "separate" twice in the same sentence.

Reply:
Agreed, updated!